poetry.lock

```diff
@@ -281,7 +281,7 @@ description = "IPython: Productive Interactive Computing"
 name = "ipython"
 optional = false
 python-versions = ">=3.6"
-version = "7.11.0"
+version = "7.11.1"
```

Comment:
A question about poetry -- right now it's set to get current declared *or newer*. Should we fix this to equal versions once in production? I imagine so.

Reply:
I would vote yes to that.